### ISSUE TITLE
F0221 18:59:03.658958  3906 blob.cpp:115] Check failed: data_

### ISSUE THREAD

**Issue by YsSue** [daijifeng001/MNC#43]
Hi, could you help me please?
When I  fine-tuning the model with:
`experiments/scripts/mnc_5stage.sh `
then there is an error after the Network initialization 
`F0221 18:59:03.658958  3906 blob.cpp:115] Check failed: data_`

# Error messages
`I0221 18:58:46.251495  3906 net.cpp:283] Network initialization done.
I0221 18:58:46.252101  3906 solver.cpp:60] Solver scaffolding done.
Loading pretrained model weights from data/imagenet_models/VGG16.mask.caffemodel
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:605] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:82] The total number of bytes read was 1024780411
I0221 18:59:01.514374  3906 upgrade_proto.cpp:67] Attempting to upgrade input file specified using deprecated input fields: data/imagenet_models/VGG16.mask.caffemodel
I0221 18:59:01.514454  3906 upgrade_proto.cpp:70] Successfully upgraded file specified using deprecated input fields.
W0221 18:59:01.514470  3906 upgrade_proto.cpp:72] Note that future Caffe releases will only support input layers and not input fields.
I0221 18:59:01.537618  3906 net.cpp:761] Ignoring source layer rpn_conv/3x3
I0221 18:59:01.537744  3906 net.cpp:761] Ignoring source layer rpn_relu/3x3
I0221 18:59:01.537762  3906 net.cpp:761] Ignoring source layer rpn/output_rpn_relu/3x3_0_split
I0221 18:59:01.667882  3906 net.cpp:761] Ignoring source layer drop6
I0221 18:59:01.680392  3906 net.cpp:761] Ignoring source layer drop7
I0221 18:59:01.761777  3906 net.cpp:761] Ignoring source layer drop6_mask
I0221 18:59:01.777510  3906 net.cpp:761] Ignoring source layer drop7_mask
Solving...
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:152: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  cur_inds = npr.choice(cur_inds, size=cur_rois_this_image, replace=False)
/media/G/yangshu/MNC/tools/../lib/transform/bbox_transform.py:201: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  bbox_targets[ind, start:end] = bbox_target_data[ind, 1:]
/media/G/yangshu/MNC/tools/../lib/transform/bbox_transform.py:202: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  bbox_inside_weights[ind, start:end] = cfg.TRAIN.BBOX_INSIDE_WEIGHTS
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:190: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  gt_box = scaled_gt_boxes[gt_assignment[val]]
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:193: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  gt_mask = gt_masks[gt_assignment[val]]
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:194: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  gt_mask_info = mask_info[gt_assignment[val]]
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:195: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  gt_mask = gt_mask[0:gt_mask_info[0], 0:gt_mask_info[1]]
/media/G/yangshu/MNC/tools/../lib/pylayer/proposal_target_layer.py:201: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  top_mask_info[i, 0] = gt_assignment[val]
F0221 18:59:03.658958  3906 blob.cpp:115] Check failed: data_ 
*** Check failure stack trace: ***
./experiments/scripts/mnc_5stage.sh: line 35:  3906 Aborted                 (core dumped) ./tools/train_net.py --gpu ${GPU_ID} --solver models/${NET}/mnc_5stage/solver.prototxt --weights ${NET_INIT} --imdb ${DATASET_TRAIN} --iters ${ITERS} --cfg experiments/cfgs/${NET}/mnc_5stage.yml ${EXTRA_ARGS}`

**Comment by SpringMai** [comment]
@YsSue I have a same problem. How did you solve it?
